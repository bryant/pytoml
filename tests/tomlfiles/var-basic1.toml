ip = "1.1.1.1"
IP = "4.4.4.4"  # This is OK - case-sensitive
