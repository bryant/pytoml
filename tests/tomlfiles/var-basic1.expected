{'ip': '1.1.1.1', 'IP': '4.4.4.4'}